schema/google/showcase/v1beta1/messaging.proto

```diff
@@ -317,6 +320,8 @@ message Blurb {
   google.protobuf.Timestamp update_time = 6 [
     (google.api.field_behavior) = OUTPUT_ONLY
   ];
+  string legacy_room_id = 7;
+  string legacy_user_id = 8;
```

Comment:
Do you think these should be wrapped in a `oneof legacy_id`? That would avoid any confusion around what is supposed to happen if both are set. 

I'm not too worried about this case though, just an idea. FWIW it will change the implementation in the server/tests a bit (`oneof` fields are represented kind of weirdly in Go).

Reply:
Yes, I think it makes more sense to combine the two new added fields as oneof. It will be clearer for the users.
So for `name` and `user` fields in `Blurb`, one represents chat room resource name, and one as author (user), should we also make them as oneof? Thanks

Reply:
> So for name and user fields in Blurb, one represents chat room resource name

Yeah the comments say something to that affect, but the server doesn't actually set the `user` field. I think that is there as an example of using a cross-proto resource_reference (User is defined in identity.proto). 

> should we also make them as oneof

Let's not change that.

Reply:
sounds cool, changed the legacy_id to be `oneof` field. Thanks